lib/internal/bootstrap/pre_execution.js

```diff
@@ -73,6 +73,13 @@ function prepareMainThreadExecution(expandArgv1 = false) {
   assert(!CJSLoader.hasLoadedAnyUserCJSModule);
   loadPreloadModules();
   initializeFrozenIntrinsics();
+
+  if (getOptionValue('--expose-primordials')) {
+    process.emitWarning(
```

Comment:
if there’s going to be this warning, should the flag name be scarier?

Reply:
I mean, all the `--expose-` stuff is around the same level right?

Reply:
yeah, i suppose that’s fair. Do they all have the same kind of emitted warning?

Reply:
> Do they all have the same kind of emitted warning?

The warning here is inspired from 

https://github.com/nodejs/node/blob/2ba8728b9f3c80a8625ec72fb7f2ccb9179d940a/lib/internal/test/binding.js#L3-L5

Reply:
`--expose-internals` has no warning but it likely should.